api/src/test/java/com/nicknathanjustin/streamercontracts/contracts/ExpiredContractsSqsHandlerTest.java

```diff
@@ -27,30 +25,30 @@
     @InjectMocks private ExpiredContractsSqsHandler expiredContractsSqsHandler;
 
     @Test(expected = NullPointerException.class)
-    public void settleExpiredDonations_nullInput_throwsException() {
-        expiredContractsSqsHandler.settleExpiredDonations(null);
+    public void settleContracts_nullInput_throwsException() {
```

Comment:
nit:

I think tests read best when its:

methodGettingTested_condtions_expectedResult

so we would name this settleAndExpireContracts_nullInput_throwsException